pkg/bootstrap/docker/run/run.go

```diff
@@ -112,7 +125,11 @@ func (h *Runner) Env(env ...string) *Runner {
 
 // Privileged tells Docker to run the container as privileged
 func (h *Runner) Privileged() *Runner {
```

Comment:
privileged + host user namespace combo works.  It may be safe here that if this method is called always call `HostUserns`.  Not setting `h.hostConfig.Privileged` is going to result in different behavior so it should still be set.